.codespell.rc

```diff
@@ -1,3 +1,3 @@
 [codespell]
-ignore-words-list = crate
+ignore-words-list = crate,sysinfo
```

Comment:
Could you add this to one of the cspell disctionary files? There should be one with a lot of crate names.

Reply:
I didn't know about cspell and I just followed method from google. I will check it out.